README.md

```diff
@@ -70,6 +104,41 @@ Attach the CRX package to the release on Github and merge back to `master`.
 ## Specifying CRX Host/Port
 
 The CRX host and port can be specified on the command line with:
-mvn -Dcrx.host=otherhost -Dcrx.port=5502 <goals>
 
+```
+mvn -Dcrx.host=otherhost -Dcrx.port=5502 <goals>
+```
 
+## Available Tags
+
+Besides the out-of-the-box tags, two new tags have been incorporated to the health check:
+### Custom tags
+
+|       Tag       | Health Checks    | Description |
+|:---------------:|------------------|-------------|
+| Shallow, Devops | SmokeHealthCheck | This tag runs a simple Smoke health check that determines if an instance is ready to serve the requests.                                                                                                                                                                                                                                          |
```

Comment:
> determines if an instance is ready to serve the requests. 

That's not entirely true. Some bundles necessary for a request to be processed may not be active and this check would return OK.